sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_utils.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from typing import cast
+from xml.etree.ElementTree import ElementTree
+
+
+import urllib.parse as urlparse
+
+from azure.servicebus.management import _constants as constants
+from ...management._handle_response_error import _handle_response_error
+
+
+async def extract_data_template(feed_class, convert, feed_element):
+    deserialized = feed_class.deserialize(feed_element)
+    list_of_qd = [convert(x) if convert else x for x in deserialized.entry]
+    next_link = None
+    if deserialized.link and len(deserialized.link) == 2:
+        next_link = deserialized.link[1].href
+    return next_link, iter(list_of_qd)
+
+
+async def get_next_template(list_func, *args, **kwargs):
+    if args:
+        next_link = args[0]
+    else:
+        next_link = kwargs.pop("next_link")
+
+    start_index = kwargs.pop("start_index", 0)
+    max_page_size = kwargs.pop("max_page_size", 100)
+    api_version = constants.API_VERSION
+    if next_link:
+        queries = urlparse.parse_qs(urlparse.urlparse(next_link).query)
+        start_index = int(queries['$skip'][0])
```

Comment:
should these be constified?  If this is the only place they're used I wouldn't feel as strongly about this/wouldn't want to muddy that shared file, but it's my immediate thought whenever I see "magic strings"

Reply:
Yes it is only used in the _utils.py files of sync and async. However I changed to use constants.